ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-authorization.j2

```diff
@@ -13,4 +13,8 @@ no aaa authorization config-commands
 {%     if aaa_authorization.commands.all_default is arista.avd.defined %}
 aaa authorization commands all default {{ aaa_authorization.commands.all_default }}
 {%     endif %}
+{%     if aaa_authorization.commands.privilege is arista.avd.defined %}
+{%         set aaa_authorization_commands_commands_console_cli = "aaa authorization commands " ~ aaa_authorization.commands.privilege.level ~ " default group " ~ aaa_authorization.commands.privilege.group %}
+{%     endif %}
+{{ aaa_authorization_commands_commands_console_cli }}
```

Comment:
This should be inside above if condition.

Reply:
Thanks that fixed it.  Odd that the build error showed up here but not on a local make converge.

Reply:
It won't because it fails at other molecule tests. If you had run ```make refresh_facts``` from molecule directory then you would have got the error.

Reply:
You need to add this to README.md and update the config in documentation template.